scripts/release/create_chart_release_pr.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+from github import Github, GithubException
+
+from lib.base_logger import logger
+
+REPO_URL = "https://github.com/mongodb/helm-charts.git"
+REPO_NAME = "mongodb/helm-charts"
+TARGET_CHART_SUBDIR = "charts/mongodb-kubernetes"
+BASE_BRANCH = "main"
+
+
+# run_command runs the command `command` from dir cwd
+def run_command(command, cwd=None):
```

Comment:
nit: please add type hints, especially since we implicitly expect an array here (looking at the logging strings)
I think subprocess accepts both strings and arrays now, but enforcing one is fine